doc/source/new-project-guide/level-4-tutorial.rst

```diff
@@ -342,6 +342,46 @@ Assume that you have successfully followed the previous steps. Now, you want to
 
 #. Done!
 
+Example usage
+^^^^^^^^^^^^^
+
+Below is a summary of the commands used at level 4 ``system`` in ``scikit-package``. See above for detailed instructions of each command.
+
+.. code-block:: bash
+
+    conda create -n skpkg_env scikit-package pre-commit
```

Comment:
for consistency across `docs` and `manusciprt`, 

should we call i `skpkg-env` or `skpkg_env`? our naming convention is that we use hyphen `-` since it's going to be a folder created.